insurance/README.md

```diff
@@ -50,13 +50,13 @@
        - AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
    - **If your data is in SQL:**
      - Add the following environment variables with your secret values in configuration tab
-       - DBUSERNAME, DBPASSWORD, DBHOSTNAME, DATABASENAME    
+       - DBUSERNAME, DBPASSWORD, DBHOSTNAME(should be specified in following format ip:port or domain:port), DATABASENAME    
```

Comment:
You can expand this into multiline as follows?

- DBUSERNAME
- DBPASSWORD
-

Reply:
okay